sanity_check.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+BESSCTL='./bessctl/bessctl'
+SCRIPTS='./bessctl/conf/samples'
+OUTFILE='sanity_check.out'
+
+TESTS='exactmatch.bess
```

Comment:
why not run everything in bessctl/conf/samples that doesn't require bringing up physical ports?